### PR TITLE
Util todo

### PR DESCRIPTION
**Pull request by GRagavi**
Please check

### CODE REVIEW

File: tests/DeleteTodo.spec.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    const id = testInfo['id']
+    const res = await deleteTodo(request,id)
+    expect(res).not.toBe(null)
+    expect(res).toBe(200)
+    })
+})
+
+
+test("Deletion of non existing todo should give 404",async ({request})=>{
+    const id = 34
```
Comment (atmnksd) [inline tests/DeleteTodo.spec.ts]: This id may exist in db and this test case may fail, instead use id 0

File: tests/GetTodo.spec.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        testInfo['id'] = body.id
+    })
+    test("get existing todo should work",async({request},testInfo)=>{
+        const id = testInfo['id']
+        const {status,body} = await getTodo(request,id)
+        expect(body).not.toBe(null)
+        expect(status).toBe(200)
+    })
+
+    test("delete created",async({request},testInfo)=>{
```
Comment (atmnksd) [inline tests/GetTodo.spec.ts]: this should be after hook and not test.

File: tests/GetTodo.spec.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+            for(let i=0;i<6;i++){
+                const {status,body} = await createTodo(request,{title:"title and status as body",status:"ACTIVE"})
+                const ID = body.id
+                id[i]= ID
+            }
+        }) 
+        test("Get Multiple Todo's",async({request},testInfo)=>{
+            const{status,body}= await getTodoAll(request)
+
```
Comment (atmnksd) [inline tests/GetTodo.spec.ts:55]: there are no assertions here.

File: tests/GetTodo.spec.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+test("get non existing todo should work",async({request},testInfo)=>{
+    const id = 34
+    const {status,body} = await getTodo(request,id)
+    expect(body).not.toBe(null)
+    expect(status).toBe(404)
+})
+
+test.describe("create and Get Multiple todo",()=>{
+    var id:number[] = new Array(5);
+        test("Create Multiple Todo's",async({request},testInfo)=>{
```
Comment (atmnksd) [inline tests/GetTodo.spec.ts]: this should be before hook and not tests

File: tests/GetTodo.spec.ts [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+                const {status,body} = await createTodo(request,{title:"title and status as body",status:"ACTIVE"})
+                const ID = body.id
+                id[i]= ID
+            }
+        }) 
+        test("Get Multiple Todo's",async({request},testInfo)=>{
+            const{status,body}= await getTodoAll(request)
+
+        })
+})
```
Comment (atmnksd) [inline tests/GetTodo.spec.ts:65]: there is no after hook for deleting multiple todos created.

File: util/todo.ts [hunk 6]
```diff
@@ -13,6 +13,40 @@ export async function createTodo(request:APIRequestContext,body:{title:string,st
...
+}
+
+export async function putTodo(request:APIRequestContext,body:{title?:string,status?:string}){
+    const respons = await request.put("/v1/todo",{
+        data: body,
+        headers:{
+         'content-Type': 'application/json'
+        }
+    })
+    return {status:respons.status(),body: await respons.json()}
```
Comment (atmnksd) [inline util/todo.ts]: Where this two functions are used? if your peer is working on this, she should add those not you.

File: tests/GetTodo.spec.ts [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+test("get non existing todo should work",async({request},testInfo)=>{
+    const id = 34
+    const {status,body} = await getTodo(request,id)
+    expect(body).not.toBe(null)
+    expect(status).toBe(404)
+})
+
+test.describe("create and Get Multiple todo",()=>{
+    var id:number[] = new Array(5);
```
Comment (atmnksd) [inline tests/GetTodo.spec.ts]: this should not be used here it should be shared with testinfo

**Review by atmnksd — CHANGES_REQUESTED** [review]

File: util/todo.ts (hunk 6 above)
Comment (GRagavi) [reply util/todo.ts]: yes.. she would be working. I just created unknowingly

File: tests/GetTodo.spec.ts (hunk 7 above)
Comment (GRagavi) [reply tests/GetTodo.spec.ts]: ok

File: tests/GetTodo.spec.ts (hunk 5 above)
Comment (GRagavi) [reply tests/GetTodo.spec.ts:65]: I shall create and update

File: tests/GetTodo.spec.ts (hunk 4 above)
Comment (GRagavi) [reply tests/GetTodo.spec.ts]: ok.. I will create it as beforehook

File: tests/GetTodo.spec.ts (hunk 3 above)
Comment (GRagavi) [reply tests/GetTodo.spec.ts:55]: will add it

File: tests/GetTodo.spec.ts (hunk 2 above)
Comment (GRagavi) [reply tests/GetTodo.spec.ts]: ok..will correct it

File: tests/DeleteTodo.spec.ts (hunk 1 above)
Comment (GRagavi) [reply tests/DeleteTodo.spec.ts]: ok sure

**Pull request merged**